doc/sphinx/addendum/program.rst

```diff
@@ -98,10 +98,10 @@ coercions.
 .. flag:: Program Mode
 
    Enables the program mode, in which 1) typechecking allows subset coercions and
-   2) the elaboration of pattern matching of :cmd:`Program Fixpoint` and
-   :cmd:`Program Definition` act
-   like Program Fixpoint/Definition, generating obligations if there are
-   unresolved holes after typechecking.
+   2) the elaboration of pattern matching of :cmd:`Fixpoint` and
+   :cmd:`Definition` act like :cmd:`Program Fixpoint`, respectively
+   :cmd:`Program Definition`, generating obligations if there are unresolved
```

Comment:
```suggestion
   :cmd:`Program Definition`respectively, generating obligations if there are unresolved
```
Also remove "respectively" on the previous line.

Reply:
Tweaked a bit more, but I can follow your proposal more closely.

Reply:
Good enough.